stages/02_process_embl.py

```diff
@@ -0,0 +1,12 @@
+from Bio import SeqIO
+import pandas as pd
+import pickle
+
+new_embl_records = SeqIO.parse("./download/miRNA.dat", "embl")
+records = [{'name': record.name, 'description': record.description,
+            'AC': record.id, 'SQ': pickle.dumps(record.seq),
```

Comment:
Can we use `str(record.seq)` here <https://biopython.org/docs/1.75/api/Bio.Seq.html#Bio.Seq.Seq.__str__>?